### PR TITLE
New language: properties configuration files

### PR DESCRIPTION
**Pull request by RaffaeleSgarro**
I provided hightlight for Java property files

```
# example
foo.bar.baz=1
```

### CODE REVIEW

File: src/languages/properties.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Category: config
+Description: Property files used in Java. A simple key value pair separated by '='
+*/
+function(hljs) {
+  return {
+    contains: [
+      hljs.HASH_COMMENT_MODE,
+      {
+        className: 'variable',
+        begin: /^[^=]+/,
```
Comment (error418) [inline src/languages/properties.js]: Keys and values can also be separated by the character `:`.
See https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html

Comment (RaffaeleSgarro) [inline src/languages/properties.js]: Somehow the pull request broke the build and so won't be merged as-is. Can't find the failing test, though, so I'm not sure I want to keep on trying

Comment (error418) [inline src/languages/properties.js]: If you don't mind I would try to implement this in a separate pull request

**Comment by paladox** [comment]
Can be abandoned as properties support was added in https://github.com/highlightjs/highlight.js/commit/d65651b8062fbc03f85fe4f73b17a02855f3b702

**Comment by egor-rogov** [comment]
Close as already implemented.

**Pull request closed without merging**